src/content/guides/spot-instances-with-on-demand-fallback/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Using on-demand instances as fall-back when spot instances are unavailable on AWS
+description: When using spot instances in a node pool on AWS, it can happen that the node pool cannot be scaled up as not enough spot instances are available. This guide shows you how to configure cluster-autoscaler in a way to provide on-demand instances as a back-up automatically.
+date: "2020-07-01"
+type: page
+weight: 120
+tags: ["tutorial"]
+---
+
+# Using on-demand instances as fall-back when spot instances are unavailable
+
+As of release v{{% first_aws_spotinstances_version %}} on AWS, node pools can use [spot instances](/basics/nodepools/#on-demand-spot). Users can select for each node pool which percentage of nodes should be covered by spot instances and by on-demand instances respectively. And users can define a base amount of on-demand instances that will be used, to ensure at least a certain amount of worker nodes is available at any time.
```

Comment:
to ensure at least a certain amount of worker nodes are available at any time.

Reply:
Modified, thanks